### DISCUSSION TITLE
Proposal to run "npm audit fix" after installation

### DISCUSSION THREAD

**Discussion by AntonOnyshch** [element-plus/element-plus#3771] [Q&A]
Getting started with command
`npm install element-plus --save`
shows me proposal to run `npm audit fix`
What does it mean?

```
added 11 packages, and audited 297 packages in 18s

39 packages are looking for funding
  run `npm fund` for details

20 high severity vulnerabilities

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

**My package.json**

```
  "devDependencies": {
    "@parcel/transformer-vue": "^2.0.0-rc.0",
    "@vue/compiler-sfc": "^3.2.19",
    "vue": "^3.2.19"
  },
  "dependencies": {
    "element-plus": "^1.1.0-beta.19"
  }
```

**Comment by jw-foss** [comment]
@AntonOnyshch Sorry for the late updates, we have turned off our dependabot for updating the dependencies since it would be much more annoying. As you mentioned we need to find out the vulnerable packages that we depended on and upgrade it cc @sxzz .

**Comment by sxzz** [comment]
These vulnerable packages are all dev dependencies, which does not seem to do any harm to the production environment.

**Reply by sxzz** [reply]
Also only `npm` will emit dev deps vulnerabilities. Just ignore it.
